web/styles/app_components.css

```diff
@@ -938,7 +938,7 @@ div.overlay {
     }
 }
 
-.zulip-icon-lock {
+.zulip-icon-lock:not(.dropdown-list .zulip-icon-lock) {
```

Comment:
`not` selectors tend to be fragile; let's just declare a separate `.dropdown-list .zulip-icon-lock` rule with the intended value.

Reply:
Done!

I used the exact same selector as the original rule for 2px `margin-top` for dropdown lock icons for clarity and specificity